### PR TITLE
Added contributions section to repository readme file

### LINKED ISSUE #6110: Proposal to add mentor-available label for open issues
I would like to propose adding a "mentor-available" label for use with issues. This would be in addition to the package-*  level labels. It would create positive reinforcement for contributors, especially for new contributors, or those new to astronomy, who are interested in being a part of the project. 

Mentors would not necessarily have to be package maintainers, but they would agree to provide a positive and guiding experience for contributors tackling the issue that they have agreed to mentor.

### PR DESCRIPTION
**Pull request by sosey**
A new contributions and feedback section was added to the master readme. Feedback on wording is encouraged! (most of wording already copied from contributions page) 

This closes #6110

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -41,6 +41,18 @@ Project Status
 For an overview of the testing and build status of all packages associated
 with the Astropy Project, see http://dashboard.astropy.org.
 
+
+Contributing Code, Documentation or Feedback
```
Comment (pllim) [inline README.rst]: Oxford comma!

Comment (sosey) [inline README.rst]: <Khaaan!!!>

**Comment by bsipocz** [comment]
@pllim - I'd leave the review to the committee.

**Comment by astrofrog** [comment]
I think it might make sense here to refer to the CONTRIBUTING.md file? (which is a standard file that GitHub recognizes and puts a link to in issues/pull requests).

**Comment by mhvk** [comment]
Ha, I learn something new every day! So, we do have a nice file, but at least if one just goes to https://github.com/astropy/astropy it is not at all obvious that this exists. So, we still should do something! @astrofrog: is your suggestion to change this PR to just add a single line to `README.rst` that links to `CONTRIBUTING.md`?

**Comment by mhvk** [comment]
I think a text like that is great; in the end, it is all about making it as easy as possible to find.

**Comment by sosey** [comment]
If the updated text listed above is approved, I can submit  an update before I leave for vacation in a couple days..... @astrofrog ?

**Comment by pllim** [comment]
Is it possible to wrap this up today before vacation and freeze?

**Comment by mwcraig** [comment]
I really like the message this addition sends; right now the README does nothing to encourage or guide people towards contributing. The edited version in https://github.com/astropy/astropy/pull/6155#issuecomment-307266901 LGTM.

**Comment by sosey** [comment]
I'm now on vacation without a computer, someone at home might want to finish this up and submit for the release?  @eteq @mwcraig feel free to update the text and push

**Comment by mwcraig** [comment]
@sosey -- enjoy your vacation! I'll push the changes.

**Comment by bsipocz** [comment]
@mwcraig - I've cancelled CI on this as the affected file is not included in any testing or docs build.

**Comment by mwcraig** [comment]
Thanks, should have included [ci skip]

**Comment by astrofrog** [comment]
@sosey - can you rebase this?

**Comment by sosey** [comment]
will do

**Comment by sosey** [comment]
@astrofrog had to remake the branch locally, take a look at the rebase and make sure it's what you expect?

File: README.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ with the Astropy Project, see http://dashboard.astropy.org.
     :alt: Powered by NumFOCUS
 
 
-Contributing Code, Documentation or Feedback
+Contributing Code, Documentation, or Feedback
```
Comment (mwcraig) [inline README.rst]: 💯  Long live the Oxford comma!

File: README.rst [hunk 3]
```diff
@@ -45,6 +45,20 @@ with the Astropy Project, see http://dashboard.astropy.org.
     :target: http://numfocus.org
     :alt: Powered by NumFOCUS
 
+
+Contributing Code, Documentation, or Feedback
+--------------------------------------------
```
Comment (pllim) [inline README.rst]: The `----` needs to be of same length as the section title.

**Comment by bsipocz** [comment]
Can you squash though, no need for 4 commits for this. Also, [skip ci] can be used, as the top level files are not tested anyway.

**Comment by astrofrog** [comment]
@bsipocz - I think we don't need to squash here as there are multiple contributors

**Pull request merged**